amazon-connect/callstats-amazon-connect-shim.js

```diff
@@ -1,4 +1,4 @@
-/*! callstats Amazon SHIM version = 1.2.0 */
+/*! callstats Amazon SHIM version = 1.2.1 */
```

Comment:
Unrelated to this PR, but why is the word "SHIM" capitalized like that everywhere? It's not an acronym...